content/pages/sc24.html

```diff
@@ -0,0 +1,70 @@
+<html>
+<head><title>sc24</title></head>
```

Comment:
`sc24` -> `SC24` ?

Reply:
all others have been lowercase... i don't think this actually renders anywhere...   look at https://irods.org/sc23 

resolving.

Reply:
`<title>` elements are rendered in the tab name.

Looking at the HTML for SC23's page, it puts "iRODS" in the tab name as defined by the `<title>` element.

Unless the tool used to publish the page modifies the HTML, I'm betting `sc24` may appear in the tab name.